neuralprophet/forecaster.py

```diff
@@ -3021,13 +3006,17 @@ def _reshape_raw_predictions_to_forecst_df(self, df, predicted, components):
                         df_forecast = pd.concat([df_forecast, yhat_df], axis=1, ignore_index=False)
         return df_forecast
 
-    def conformalize(self, df_cal, alpha, method="naive", plotting_backend="default"):
-        """Apply a given conformal prediction technique to get the uncertainty prediction intervals (or q-hats).
+    def conformal_predict(
+        self, df, calibration_df=None, alpha=0.1, method="naive", plotting_backend="default", **kwargs
```

Comment:
Not sure myself: Should we better explicitly define the parameters here? If people use the docstrings and code reference to understand this method they won't get far like this. 🤔